src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Cmdlets/Jobs/GetAzureRmRecoveryServicesBackupJob.cs

```diff
@@ -117,6 +117,10 @@ public override void ExecuteCmdlet()
                 {
                     throw new Exception(Resources.JobAllowedDateTimeRangeExceeded);
                 }
+                else if(rangeStart > DateTime.UtcNow)
```

Comment:
Please add automated tests for this logic.  These can be unit tests of the date validation logic or scenario tests.  There should be some automation to prevent regression on this issue in future check-ins.